addons/osio-addon/src/test/java/io/fabric8/launcher/osio/providers/DependencyParamConverterProviderIT.java

```diff
@@ -10,64 +8,28 @@
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 
-import io.fabric8.launcher.base.test.HttpApplication;
-import io.restassured.builder.RequestSpecBuilder;
-import io.restassured.specification.RequestSpecification;
 import org.apache.maven.model.Dependency;
-import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.container.test.api.RunAsClient;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.test.api.ArquillianResource;
-import org.jboss.shrinkwrap.api.Archive;
-import org.jboss.shrinkwrap.api.ShrinkWrap;
-import org.jboss.shrinkwrap.api.asset.FileAsset;
-import org.jboss.shrinkwrap.api.spec.WebArchive;
-import org.jboss.shrinkwrap.resolver.api.maven.Maven;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
 import static io.restassured.RestAssured.given;
 import static java.util.stream.Collectors.joining;
 import static org.hamcrest.Matchers.allOf;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.isEmptyString;
-import static org.jboss.shrinkwrap.resolver.api.maven.ScopeType.COMPILE;
 
 /**
  * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
  */
-@RunWith(Arquillian.class)
-@RunAsClient
+@Disabled
```

Comment:
that is a bit sad

Reply:
I forgot to enable this again. Good catch!

Reply:
Ah no, this is OSIO and it's going to be removed in https://github.com/fabric8-launcher/launcher-backend/pull/690, nevermind